applications/CoSimulationApplication/python_scripts/data_transfer_operators/kratos_mapping.py

```diff
@@ -12,10 +12,20 @@
 # other imports
 from KratosMultiphysics.CoSimulationApplication.utilities import data_communicator_utilities
 from time import time
+from collections import namedtuple
+
+# Define a named tuple for clarity
+SolverData = namedtuple('SolverData', [
+    'model_part_origin', 'model_part_destination',
+    'model_part_origin_name', 'model_part_destination_name',
+    'variable_origin', 'variable_destination',
+    'mapper_flags',
+    'identifier_origin', 'identifier_destination',
+    'identifier_tuple', 'inverse_identifier_tuple'
+])
```

Comment:
please use a `dataclass`  for this => https://docs.python.org/3/library/dataclasses.html